spec/50-privacy.md

```diff
@@ -14,7 +14,7 @@ Note, both privacy concerns of `traceparent` field are theoretical rather than p
 
 ## Privacy of tracestate field
 
-The  `tracestate` field may contain any opaque value in any of the keys. The main purpose of this header is to provide additional vendor-specific trace-identification information across different distributed tracing systems. 
+The  `tracestate` field may contain any opaque value in any of the keys. The main purpose of this header is to provide additional vendor-specific trace-identification information across different distributed tracing systems.
```

Comment:
Extra space between "The" and "`tracestate`".

Reply:
Fixed